sw/device/lib/testing/check.h

```diff
@@ -53,33 +53,41 @@
  * Prints differences between `actual_` and `expected_` before assigning the
  * comparison `result_` value.
```

Comment:
nit: I think this line should be updated (log and abort).

Reply:
Addressed in #9067.